config/karma.js

```diff
@@ -0,0 +1,16 @@
+var grunt = require('grunt');
+var testOption = grunt.option('tests');
```

Comment:
I was going to. The recommended way that the webpack tests are setup means the tests are bundled. This is the fastest way to have tests running. I'm going to start with that and come back to dealing with the single test case if it comes up later.